spec/index.md

```diff
@@ -329,10 +328,10 @@ If the app needs to authenticate the identity of the end-user, it should
 include two OpenID Connect scopes:  `openid` and `profile`.   When these scopes
 are requested, and the request is granted, the app will receive an id_token
 along with the access token.  For full details, see [SMART launch context
-parameters](./scopes-and-launch-context).
+parameters](scopes-and-launch-context/index.html).
 
 #### *For example*
-An app that needs demographics and observations for a single
+If an app that needs demographics and observations for a single
```

Comment:
I see where we agreed to add "If" in https://github.com/HL7/smart-app-launch/issues/90 -- but reading this in context, I think we also need to delete "that", so this reads "If an app needs demographics..."